src/Logger.hs

```diff
@@ -20,7 +20,5 @@ data LogEntry =
   | LogEntryForModerator EventLogItemCold
   deriving (Eq, Show)
 
-type SendLogMsg = LogEntry -> IO ()
```

Comment:
well, that just means those constructions should be eliminated as well...  :-)

hm....  i'm thinking...